toolbox.py

```diff
@@ -396,3 +405,55 @@ def identifyObtrusiveness(blockStr):
     if blockStr in lookup.OBTRUSIVE:
         return 3
     return 4
+
+
+def visualizeHeightmap(*arrays, title=None, autonormalize=True):
+    """**Visualizes one or multiple numpy arrays**.
+
+    Args:
+        title (str, optional): display title. Defaults to None.
+        autonormalize (bool, optional): Normalizes the array to be
+            between 0 (black) and 255 (white). Defaults to True.
+    """
+    def normalize(array):
+        """**Normalize the array to contain values from 0 to 1**."""
+        return (array - array.min()) / (array.max() - array.min())
+
+    for array in arrays:
+        if autonormalize:
+            array = (normalize(array) * 255).astype(np.uint8)
+
+        plt.figure()
+        if title:
+            plt.title(title)
+        plt_image = cv2.cvtColor(array, cv2.COLOR_BGR2RGB)
+        plt.imshow(plt_image)
+    plt.show()
+
+
+# ========================================================= converters
+# The 'data types' commonly used in this package are:
+#
+# facing: any of 'north', 'east', 'south', 'west' (can be a sequence)
+# rotation: value from 0-15 representing number of clockwise increments of pi/4
+#
+# axis: 'x', 'y', 'z'
+# vector: any multiple of (1, 1, 1) e.g. (0, 1, -4)
+# =========================================================
```

Comment:
Less complicated converters are directly mapped in `lookup.py`. Perhaps some wrapper functions could/should be made and placed here, or maybe this comment should be removed/reduced

Reply:
I think having the wrapper functions makes sense. Because people won't know to look for them in lookup.py, and it's good to have them all in one place. Also using a function is much more intuitive that a dictionary.
Not sure what this really has to do with the comment. The comment is good, assuming people will find it here.